src/platform/lumin-runtime/elements/builders/button-builder.js

```diff
@@ -4,7 +4,7 @@ import { ui } from 'lumin';
 
 import { TextContainerBuilder } from './text-container-builder.js';
 import { ArrayProperty } from '../properties/array-property.js';
-import { PrimitiveTypeProperty } from '../properties/primitive-type-property.js';
```

Comment:
what happened to PrimitiveTypeProperty?  We don't need it for button anymore?

Reply:
PrimitiveTypeProperty was not used before that change. I used the opportunity to cleanup.